task/06-conditions-n-loops-tasks.js

```diff
@@ -192,7 +220,11 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+    const left = isStartIncluded ? "[" : "(";
```

Comment:
`left` and `right`? What do you mean?)) Use more meaningful names for variables.